test/powershell/Modules/Microsoft.PowerShell.Management/Get-ComputerInfo.Tests.ps1

```diff
@@ -341,11 +341,16 @@ public static extern int LCIDToLocaleName(uint localeID, System.Text.StringBuild
         $returnValue = $null
```

Comment:
Well, if the failure prompting all of this was Nano or IoT, that might solve the immediate namespace-not-found issue, but there will still likely be test failures.

On one of my machines, for example, Get-CimInstance successfully retrieves the WMI object, but Get-ComputerInfo sets the DeviceGuard properties to null because the licensing property "CodeIntegrity-AllowConfigurablePolicy" is not properly set. This results in `Should Be` failures.

I'm proposing that the tests behave in a manner consistent with how the cmdlet is designed to work.